docs/index.md

```diff
@@ -130,3 +130,43 @@ napari is extensible! Find plugins, or develop your own!
 :::
 
 ::::
+
+## funding
```

Comment:
I assume we are using lowercase on purpose and I am not opposed to it for consistency with the title, but flagging just in case 😄